src/yang/iana-qos-types.yang

```diff
@@ -0,0 +1,522 @@
+module iana-qos-types {
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:iana-qos-types";
+  prefix qos-types;
+
+  organization
+    "IETF Routing Area Working Group";
+
+  contact
+    "WG Web:   <https://datatracker.ietf.org/wg/rtgwg/>
+     WG List:  <mailto:rtgwg@ietf.org>
+
+     Editor:   Aseem Choudhary
+               <mailto:asechoud@cisco.com>
+     Editor:   Mahesh Jethanandani
+               <mailto:mjethanandani@gmail.com>";
+
+  description
+    "This module contains type definitions for all QoS types.
+
+     Copyright (c) 2022 IETF Trust and the persons identified as
+     authors of the code.  All rights reserved.
+
+     Redistribution and use in source and binary forms, with or
+     without modification, is permitted pursuant to, and subject
+     to the license terms contained in, the Simplified BSD License
+     set forth in Section 4.c of the IETF Trust's Legal Provisions
+     Relating to IETF Documents
+     (http://trustee.ietf.org/license-info).
+
+     This version of this YANG module is part of RFC XXXX; see
+     the RFC itself for full legal notices.";
+
+  revision YYYY-MM-DD {
+    description
+      "Initial version";
+    reference
+      "RFC XXXX: YANG Models for Quality of Service (QoS).";
+  }
+
+  /*
+   * Features.
+   */
+  feature meter-template-support {
```

Comment:
you may want to update the codebase. Most of these features were removed.

Reply:
@aseemavi , I am up-to-date with whatever is in the codebase. If you removed any features, they must be local to you.

Reply:
In latest published draft (08), there are only 5 features:
child-policy-feature
count-feature
named-counter-feature
policy-inline-classifier-config
classifier-template-feature

Rest were cleaned up. If it is part of code base and not part of latest published draft, then we have to see how it happened.

Reply:
Ok. I have removed all but the five above features from all the files. Also there are two grouping that were not being referenced by anyone. Do not know if they got missed or are no longer needed. 

Please review the changes closely to make sure we have not missed anything.